ambry-network/src/test/java/com.github.ambry.network/SSLSelectorTest.java

```diff
@@ -203,6 +202,36 @@ public void testEmptyRequest()
     assertEquals("", blockingRequest(connectionId, ""));
   }
 
+  @Test
+  public void testSSLConnect()
+      throws IOException {
+    String connectionId =
+        selector.connect(new InetSocketAddress("localhost", server.port), BUFFER_SIZE, BUFFER_SIZE, PortType.SSL);
+    Assert.assertFalse("Channel should not be ready by now (until handshake completes)",
+        selector.isChannelReady(connectionId));
+    while (!selector.connected().contains(connectionId)) {
+      selector.poll(10000L);
+    }
+    Assert.assertTrue("Channel should have been ready by now ", selector.isChannelReady(connectionId));
+  }
+
+  @Test
+  public void testCloseDuringSslHandshake()
+      throws IOException {
+    String connectionId =
+        selector.connect(new InetSocketAddress("localhost", server.port), BUFFER_SIZE, BUFFER_SIZE, PortType.SSL);
+    Assert.assertFalse("Channel should not be ready by now (until handshake completes)",
+        selector.isChannelReady(connectionId));
+    selector.poll(10000L);
```

Comment:
not going to make any diff. We will have one key chosen that we know of. I am just being in sync with other timeouts in this test class